### PR TITLE
Added new Kernel for RHEL75

### PR DESCRIPTION
**Pull request by DivyaJyothiKasam**

### CODE REVIEW

File: ips.sh [hunk 1]
```diff
@@ -84,5 +84,6 @@ centos75x64update5="192.168.222.89"
 centos75x64update6="192.168.222.91"
 centos75x64update7="192.168.222.93"
 centos75x64update8="192.168.222.99"
+centos75x64update9="192.168.222.98"
```
Comment (prashantmhetar) [inline ips.sh:87]: Can you please share "uname -a"  output for this new VM?

Comment (DivyaJyothiKasam) [reply ips.sh:87]: [root@centos74x64 ~]# uname -a
Linux centos74x64 3.10.0-862.27.1.el7.x86_64 #1 SMP Fri Dec 14 16:31:10 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Review by prashantmhetar — APPROVED** [review]
changes looks good

**Comment by mboitei** [comment]
Merge for new Errata Kernel 3.10.0-862.27.1

**Pull request merged**